cellbase-app/app/azure/arm/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+                    },
+                    "appGwSslEmail": {
+                        "value": "[parameters('sslCertificateEmail')]"
+                    },
+                    "LogAnalyticsWorkspaceResourceID": {
+                        "value": "[parameters('LogAnalyticsWorkspaceResourceID')]"
+                    }
+                }
+            }
+        },
+        {
+            "comments": "This deploys a VMSS which hosts cellbase behind a LB with healthchecking, autoscaling and rolling update",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[concat(parameters('rgPrefix'),'-cellbase')]",
+            "apiVersion": "2018-05-01",
+            "name": "webservers",
+            "dependsOn": [
+                "cellbase-vnet",
+                "[concat(parameters('rgPrefix'),'-cellbase')]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(parameters('_artifactsLocation'), '/webservers/azuredeploy.json', parameters('_artifactsLocationSasToken'))]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "_artifactsLocation": {
+                        "value": "[parameters('_artifactsLocation')]"
+                    },
+                    "_artifactsLocationSasToken": {
+                        "value": "[parameters('_artifactsLocationSasToken')]"
+                    },
+                    "virtualMachineSize": {
+                        "value": "[variables('selectedSize').webServers.nodeSize]"
+                    },
+                    "minInstanceCount": {
+                        "value": "[variables('selectedSize').webServers.minNodeCount]"
+                    },
+                    "initialInstanceCount": {
+                        "value": "[variables('selectedSize').webServers.minNodeCount]"
+                    },
+                    "maxInstanceCount": {
+                        "value": "[variables('selectedSize').webServers.maxNodeCount]"
+                    },
+                    "virtualNetworkSubnetId": {
+                        "value": "[reference('cellbase-vnet').outputs.defaultSubnetId.value]"
+                    },
+                    "appgwBackendAddressPoolId": {
+                        "value": "[reference('appgw-ssl').outputs.appgwBackendAddressPoolId.value]"
+                    },
+                    "SSHKeyData": {
+                        "value": "[parameters('sshAdminKeyData')]"
+                    },
+                    "cellbaseContainerImage": {
+                        "value": "[parameters('cellbaseContainerImage')]"
+                    },
+                    "mongoHosts": {
+                        "value": "[replace(replace(replace(replace(string(reference('cellbase-mongodb').outputs.dnsNames.value),'[',''),'{\"fqdn\":\"',''),'\"}',''),']','')]"

```

Comment:
This is .... guessing we can't do it any other way right?

Reply:
Not that sprung to mind... Need to convert array to comma separated string. Any ideas?

Reply:
Would it be possible to index into the array instead, guessing that wouldn't work so if not then no worries

Reply:
Would have to loop through the number of elements... will have a think.